src/app/story/directives/basic.component.ts

```diff
@@ -95,12 +95,9 @@ import { HalDoc, ModalService, TabService } from 'ngx-prx-styleguide';
         to either delay or back-date its release, select the desired release date and time here.
         Otherwise, the episode will be released immediately once published.
         </div>
-        <prx-datepicker *ngIf="showReleasedAt"
+        <prx-tz-datepicker *ngIf="showReleasedAt"
```

Comment:
I notice the timezone part of this has some vertical issues, with the cursor / text / carat not all lining up.  Seems to only happen if you try to change the timezone - it starts out correctly aligned.

![image](https://user-images.githubusercontent.com/1410587/55735420-875ad700-59de-11e9-84cc-8c8b59080f3f.png)

Reply:
I also wish this field allowed null values without being marked as "Invalid Date".  (As the old date/timepickers did).  A bit jarring to click the "Specify date and time" and immediately being hit with the red outline.

Reply:
Maybe a compromise would be to explicitly set `story.releasedAt = new Date()` whenever that checkbox is checked.  And set it back to null when unchecked.